src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  pressedKey: string | null;
+}
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    pressedKey: null,
```

Comment:
Not sure this is right way to set default value for string.

Reply:
that's right